src/extension/rpc.ts

```diff
@@ -39,7 +40,7 @@ export function createRpcClient<Messages extends MessageCollection>(
 
         const timeout = setTimeout(() => {
           removeListener();
-          reject(new Error("Timeout waiting for message"));
+          reject(new Error(RPC_MESSAGE_TIMEOUT));
```

Comment:
I also added a new constant for this case to more easily compare when the timeout occurs.

We could also consider a `TimeoutError` class if we'd rather compare on the instance level rather than the message itself. For now I think this works well 🙂